cmd/build.go

```diff
@@ -55,6 +56,7 @@ and the image name is stored in the configuration file.
 	cmd.Flags().StringP("image", "i", "", "Full image name in the form [registry]/[namespace]/[name]:[tag] (optional). This option takes precedence over --registry (Env: $FUNC_IMAGE)")
 	cmd.Flags().StringP("registry", "r", GetDefaultRegistry(), "Registry + namespace part of the image to build, ex 'quay.io/myuser'.  The full image name is automatically determined based on the local directory name. If not provided the registry will be taken from func.yaml (Env: $FUNC_REGISTRY)")
 	cmd.Flags().BoolP("push", "u", false, "Attempt to push the function image after being successfully built")
+	cmd.Flags().StringP("platform", "", "", "Target platform to build (e.g. linux/amd64).")
```

Comment:
Yes, I also think that for completeness all our flags should have an associated environment variable.
While I certainly lean towards using flags in most cases, there are patterns/tools/techniques out there that perfer to specify options in locally-scoped environment variables.

For example I sometimes set up a `.direnv` in in-progress codebases so that my shell history can only be the flags which are actually changing between runs. ¯\_(ツ)_/¯